### PR TITLE
Delete improvements

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

**Comment by xitronix** [comment]
Screenshot 2019-11-12 at 16 51 51
Screenshot 2019-11-12 at 16 52 18

File: universal-login-react/src/core/services/DisconnectAccountService.ts [hunk 1]
```diff
@@ -13,30 +13,30 @@ interface InputsType {
 
 const checkInputsAgainstError = (usernameLogged: string, inputs: InputsType) => ({
   usernameError: inputs.username !== usernameLogged,
-  verifyFieldError: inputs.verifyField !== 'DELETE MY ACCOUNT',
+  verifyFieldError: inputs.verifyField !== 'DISCONNECT',
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/core/services/DisconnectAccountService.ts]: Pls, make it const and pls, use it in tests

Comment (marekkirejczyk) [reply universal-login-react/src/core/services/DisconnectAccountService.ts]: 👍

Comment (xitronix) [reply universal-login-react/src/core/services/DisconnectAccountService.ts]: ✔️

**Comment by xitronix** [comment]
Screenshot 2019-11-13 at 10 29 58
Screenshot 2019-11-13 at 10 29 51

**Comment by cehali** [comment]
Tested manually, works for me.

**Pull request merged**